iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -98,8 +101,14 @@ public static class MessageSystemPropertyNames
         /// </summary>
         public const string ConnectionDeviceGenerationId = "iothub-connection-auth-generation-id";
 
+        /// <summary>
+        /// The connection authentication method value is ignored for cloud-to-device messages.
+        /// </summary>
         public const string ConnectionAuthMethod = "iothub-connection-auth-method";
```

Comment:
can we add a note specifying what this auth method signifies?

Reply:
I have no idea about this and this is ignored for c2d messages (based on my conversation with hub team) so there is no need to add extra info.